sqlalchemy_redshift/dialect.py

```diff
@@ -255,7 +255,8 @@ def post_create_table(self, table):
                     for key in keys]
             if interleaved_sortkey:
                 text += " INTERLEAVED"
-            text += " SORTKEY ({0})".format(", ".join(keys))
+            text += " SORTKEY ({0})".format(
+                ", ".join('"' + key + '"' for key in keys))
```

Comment:
I think we can have our cake and eat it too. SQLAlchemy's [`IdentifierPreparer.quote` method](http://docs.sqlalchemy.org/en/latest/core/internals.html?highlight=identifierpreparer#sqlalchemy.sql.compiler.IdentifierPreparer.quote) exists to conditionally quote identifiers. I think we should use that here rather than putting in explicit quotes. It would be safer (there might be some edge cases we aren't considering if column names contain quotes themselves) and it means quotes aren't added to identifiers unless necessary, which would make output more natural.

So I think we could use something like:
```python
sortkey_string = ", ".join(self.preparer.quote(key) for key in keys)
text += " SORTKEY ({0})".format(sortkey_string)
```